src/nodes/display/CDLNode.js

```diff
@@ -0,0 +1,83 @@
+import TempNode from '../core/TempNode.js';
+import { Fn, nodeObject, vec3, vec4 } from '../tsl/TSLBase.js';
+import { max } from '../math/MathNode.js';
+import { LinearSRGBColorSpace } from '../../constants.js';
+import { ColorManagement } from '../../math/ColorManagement.js';
+import { Vector3 } from '../../math/Vector3.js';
+
+/**
+ * Color Decision List (CDL) v1.2
+ *
+ * References:
+ * - ASC CDL v1.2
+ * - https://blender.stackexchange.com/a/55239/43930
+ * - https://docs.acescentral.com/specifications/acescc/
+ */
+class CDLNode extends TempNode {
+
+	static get type() {
+
+		return 'CDLNode';
+
+	}
+
+	constructor( inputNode, slopeNode, offsetNode, powerNode, saturationNode ) {
```

Comment:
Is it overkill for the parameters to be Nodes? Can't they just be float (or ultimately vec3)

Reply:
Probably right – I'll make the change to just constant `float | vec3` inputs unless @sunag has other preferences for node best practices? 

We should give each a default value, as well.

Reply:
I'd really like to see them as vec3 so we can experiment.

Reply:
I don't think it would be a problem, as you are using `nodeObject()` in the `cdl()` function parameters, it can support any primitive value like a JS number, constant, uniform or procedural, float or vec3. We can let the user choose what is best for him.

Reply:
@donmccurdy Maybe you can use node args and write the code it works with either float or vec3.

Reply:
@WestLangley it does work with either float or vec3. :) 

With vec3 input we can, for example, do the "AgX Golden" look:

![chair_agx_golden](https://github.com/user-attachments/assets/ff3f8a25-1d44-41c6-b142-1bb3eea7d0c3)

Reply:
Great! That is exactly why I was pushing for vec3 support!